modules/deepbgc/download/main.nf

```diff
@@ -0,0 +1,32 @@
+process DEEPBGC_DOWNLOAD {
+    tag "download"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::deepbgc=0.1.30" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/deepbgc:0.1.30--pyhb7b1952_1':
+        'quay.io/biocontainers/deepbgc:0.1.30--pyhb7b1952_1' }"
+
+    output:
+    path "deepbgc_db"             , emit: db
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    //To specify a downloads directory different to the default: set DEEPBGC_DOWNLOADS_DIR env variable
```

Comment:
This might be confusing for a pipeline developer making it seem they are allowed to modify the module. You could instead put `from DeepBGC docs:` at the beginnign if you think this is necessary.

Reply:
Ok, I'll take it out then